client/client_test.go

```diff
@@ -208,17 +209,19 @@ func TestClient_RPC_Passthrough(t *testing.T) {
 
 func TestClient_Fingerprint(t *testing.T) {
 	t.Parallel()
+	require := require.New(t)
+	if _, ok := driver.BuiltinDrivers["mock_driver"]; !ok {
+		t.Skip(`test requires mock_driver; run with "-tags nomad_test"`)
```

Comment:
Can you add this to a `driver/testing.go` helper and replace existing uses of this with the shared helper.